### PR TITLE
Rebrand - Brand kit typography and colour updates

### PR DESCRIPTION
**Pull request by ethancroft**
Ticket: TODO
Task: https://webjobs.agepartnership.co.uk/#/tasks/14168320 (no ticket at time of writing)

Changes:
- Adds individual vars for different shades of colours
- Amends existing vars with numbered vars which match (or are closest to) existing values
- Amends letter spacing of font mixins

Checks:
- ISO checks/malice 
- Sensible changes of existing vars

Note: We'll need to change any uses of `$pale-yellow` in any repo that uses it to use the light gold variable (Done here: https://github.com/AgePartnership/mjolnir-templates/pull/1796)

### CODE REVIEW

File: rebrand/config/_typography.scss [hunk 1]
```diff
@@ -71,6 +71,6 @@ sup {
 
 @mixin fsize-36 {
     font-size: 36px !important;
-    letter-spacing: 0.02em;
+    letter-spacing: 0.08em;
```
Comment (beccadawson) [inline rebrand/config/_typography.scss:74]: Compared to the designs, I think we want to stick with 0.02 or maaaaybe 0.04? I think there's differences between how Sketch and browsers render fonts, so the letter spacing and line-height can be a bit tricky to specify!

**Review by beccadawson — CHANGES_REQUESTED** [review]

File: rebrand/config/_typography.scss (hunk 1 above)
Comment (ethancroft) [reply rebrand/config/_typography.scss]: I've fixed that - do you want me to install onto mjolnir-templates/web or do you think it's not necessary?

Comment (beccadawson) [reply rebrand/config/_typography.scss]: What's your argument for not doing? I would do for completeness, but I could be swayed!

**Review by beccadawson — APPROVED** [review]

File: rebrand/config/_typography.scss (hunk 1 above)
Comment (ethancroft) [reply rebrand/config/_typography.scss]: Now I've reverted the letter spacing change, only the colours have changed. We know the letter-spacing of that mixin is fine cos we use it on live, so really you're only just checking the colours now on the previews :)

**Pull request merged**